Source/common/SNTConfigurator.h

```diff
@@ -320,6 +320,11 @@
 ///
 @property(readonly, nonatomic) BOOL enableForkAndExitLogging;
 
+///
+///  If true, skip processing any actions from other endpoint security clients. Defaults to false.
```

Comment:
Call out that this only applies when running as a sysx; _other_ implies it but let's make it clear.

Reply:
Done